packages/edit-site/src/components/style-book/index.js

```diff
@@ -405,6 +422,41 @@ export const StyleBookPreview = ( { userConfig = {}, isStatic = false } ) => {
 	const examples = getExamples( colors );
 	const examplesForSinglePageUse = getExamplesForSinglePageUse( examples );
 
+	let previewCategory = null;
+	if ( section.includes( '/colors' ) ) {
+		previewCategory = 'colors';
+	} else if ( section.includes( '/typography' ) ) {
+		previewCategory = 'text';
+	} else if ( section.includes( '/blocks' ) ) {
+		previewCategory = 'blocks';
+		const blockName =
+			decodeURIComponent( section ).split( '/blocks/' )[ 1 ];
+		if ( blockName ) {
+			previewCategory = blockName;
+		}
+	} else if ( ! isStatic ) {
+		previewCategory = 'overview';
+	}
+	const categoryDefinition = STYLE_BOOK_PREVIEW_CATEGORIES.find(
+		( category ) => category.slug === previewCategory
+	);
+
+	// If there's no category definition there may be a single block.
+	const filteredExamples = categoryDefinition
+		? getExamplesByCategory( categoryDefinition, examples )
+		: {
+				examples: [
+					examples.find(
+						( example ) => example.name === previewCategory
```

Comment:
`comments-pagination`, `list-item` and maybe some other blocks don't have any examples and will bork the whole page. 

`/wp-admin/site-editor.php?p=%2Fstyles&section=%2Fblocks%2Fcore%252Fcomments-pagination&preview=stylebook`

`/wp-admin/site-editor.php?p=%2Fstyles&section=%2Fblocks%2Fcore%252Flist-item&preview=stylebook`

Some other blocks are listed on https://github.com/WordPress/gutenberg/issues/64707. Also there are links to PRs where we created missing block examples.

Reply:
Oh I just fixed that! Blocks without examples should now just show the all blocks page, I think that's the best we can do until we get all the examples added.